projects/github.com/bcicen/ctop/package.yml

```diff
@@ -0,0 +1,31 @@
+distributable:
+  url: https://github.com/bcicen/ctop/archive/refs/tags/v{{version}}.tar.gz
+  strip-components: 1
+
+display-name: ctop
+
+versions:
+  github: bcicen/ctop
+
+build:
+  dependencies:
+    go.dev: ^1.18
+    git-scm.org: "*"
+
+  script: |
+    go mod download
+    CGO_ENABLED="$CGO_ENABLED" go build -tags release -ldflags="$GO_LDFLAGS" -o "$BUILDLOC"
```

Comment:
`CGO_ENABLED="$CGO_ENABLED"` shouldn't do anything, unless something is very weird in the shell.